packages/utils/navigation.ts

```diff
@@ -33,7 +33,7 @@ export type RootStackParamList = {
   RiotGameFight: undefined;
   RiotGameBreeding: undefined;
   RiotGameMemories: undefined;
-  RiotGameMarketplace: undefined;
+  RiotGameMarketplace: { collectionId?: string } | undefined;
```

Comment:
this seems weird,
I think this might break navigation configuration 
Maybe just use `{ collectionId?: string }`

Reply:
actually without ```| undefined``` it will break ts check in ```navigation.navigate(item.route);``` in ```RiotGameHeader``` file